cli/src/commands/deploy.ts

```diff
@@ -0,0 +1,19 @@
+import { Command, flags } from '@oclif/command';
+
+export default class Deploy extends Command {
+  public static description = 'deploy your ShiftJS project to its associated domain';
+
+  public static examples = [
+    '$ shift-cli deploy',
```

Comment:
The prefix `$ shift-cli` should be easy to replace.
Maybe use a function to change this?
The spec asks to use `npx shift-cli`.
We might just want to see how the cli was called and use that as the prefix.

Reply:
fixed